pkg/internal/checkup/checkup_test.go

```diff
@@ -471,6 +479,14 @@ func (cs *clientStub) VMIName(namePrefix string) string {
 	return ""
 }
 
+func successfulRunResults() status.Results {
+	const sentPackets = 10
+	return status.Results{
+		TrafficGenSentPackets:      sentPackets,
+		VMUnderTestReceivedPackets: sentPackets,
+	}
+}
```

Comment:
Shouldn't this be introduced in a previous commit?

Reply:
nope. up until we introduce this condition, the empty struct can pass the unit tests, but it doesn't feel right to document it as a `successfulRunResults`, so I preferred to introduce it only after we've properly introduced the condition.